registry.k8s.io/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
why is this in this repo instead of k8s.io?

Reply:
To test image promotion to Artifact registry. Trying to replicate process without block it with experimentation.